packages/prisma-faker/src/seed.ts

```diff
@@ -909,7 +910,7 @@ export function seed<
             }
             case 'Int': {
               const number = faker.integer({
-                min: -2147483647,
+                min: 1,
```

Comment:
Negative integers don't work as ID for some stuff, since I restored `Int` ID as a workaround for BYOI, I had to make this change breaking it (in a way) for non ID Int fields!